src/app/components/drawer/index.tsx

```diff
@@ -10,21 +10,25 @@ import {
   ProfileIcon,
   SettingsIcon,
 } from 'src/assets/svg/navigation-icons/';
-import ROUTES from 'src/constants/routes';
 import { RootState, useAppDispatch } from 'src/store';
 import { startLogoutUser } from 'src/store/auth/thunks';
 
 import { MenuItem } from './components/menu-item';
 
 const navigationRoutes = [
-  { route: ROUTES.HOME, icon: <HomeIcon color="#4D7C0F" />, label: 'Home' },
   {
-    route: ROUTES.PROFILE,
+    route: 'Main',
+    params: { screen: 'Home' },
+    icon: <HomeIcon color="#4D7C0F" />,
```

Comment:
we can avoid hardcoding colors by using useTheme inside the component and passing the color as prop within the component : 
```
        <MenuItem
            key={route}
            icon={React.cloneElement(icon, {
              color: theme.palette.primary[500],
            })}
            label={label}
            onPress={() => navigation.navigate(route)}
          />
          ```

Reply:
Ehh dont burn me! Hah